manifests/v/VNGCorp/Zalo/23.7.1/VNGCorp.Zalo.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created with Komac v1.9.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: VNGCorp.Zalo
+PackageVersion: 23.7.1
+InstallerType: nullsoft
+Installers:
+- Architecture: x86
+  InstallerUrl: https://res-download-pc-te-vnso-tt-10.zadn.vn/win/ZaloSetup-23.7.1.exe

```

Comment:
```suggestion
  InstallerUrl: https://res-zaloapp-aka.zdn.vn/win/ZaloSetup-23.7.1.exe
```

Reply:
Hi @giatrung2012,

This is the InstallerUrl I find from opening the PackageUrl. Is this the same package?

Reply:
Yes, this is the same package.